avaframe/com1DFA/com1DFA.py

```diff
@@ -2132,6 +2139,9 @@ def prepareVarSimDict(standardCfg, inputSimFiles, variationDict):
             info dict on available input data
         variationDict: dict
             dictionary with parameter to be varied as key and list of it's values
+        simNameOld: list
+            list of simulation names that already exist (optional). If provided,
+            only cary on simulation that do not exist
```

Comment:
suggestion: only perform simulations that do not already exist - if not typo in carry